packit/utils/changelog_helper.py

```diff
@@ -69,10 +69,10 @@ def entry_from_action(self) -> Optional[str]:
 
     def update_dist_git(self, full_version: str, upstream_tag: str) -> None:
         """
-        Update the spec-file in dist-git by setting the 'Version' tag and
+        Update the spec-file in dist-git and set the 'Version' tag and
```

Comment:
What does "update the spec-file in dist-git" mean, besides setting Version and adding new entry to %changelog? (Is there anything else to update?)

Reply:
Syncing the whole content but changelog.